src/main/java/com/cegeka/academy/service/mapper/InvitationMapper.java

```diff
@@ -28,6 +28,24 @@ public static InvitationDTO convertInvitationEntityToInvitationDTO(Invitation in
         return invitationDTO;
     }
 
+    public static Invitation convertInvitationDTOToInvitation(InvitationDTO invitationDTO) {
+
+        Invitation invitation = new Invitation();
+        invitation.setStatus(invitationDTO.getStatus());
+        invitation.setDescription(invitationDTO.getDescription());
+        invitation.setEvent(null);
+
+        if(invitationDTO.getUserId() != null){
+
+            User newUser = new User();
+            newUser.setId(invitationDTO.getUserId());
+            invitation.setUser(newUser);
```

Comment:
Tocmai ce v-am spus acum 3 zile pe whatsapp sa nu mai setati id-urile entitatilor in DTO-uri. Daca doresti sa adaugi pe o invitate un user trebuie sa-l cauti in baza de date si sa faci set pe entitatea Invitation. In mapper convertesti doar informatiile care tin exclusiv de Invitation, DAR FARA ID